environments/base/default-config.libsonnet

```diff
@@ -34,6 +34,7 @@
     retentionResolutionRaw: '14d',
     retentionResolution5m: '1s',
     retentionResolution1h: '1s',
+    enableDownsampling: true,
```

Comment:
note: here we are changing the default downsampling behavior of all environments that inherit from `base`.

cc @observatorium/maintainers

Reply:
Can we have it merged if no further concerns? @squat

Reply:
I think we shouldn't change the default behaviour for the `base`.

Reply:
We should preferable leave the default to disabled and then if you want it, you can explicitly enable it in the custom resource when you run the operator. WDYT, @clyang82?

Reply:
Yes. Agree with that. Let me update accordingly.